riak_test/yz_pb.erl

```diff
@@ -240,7 +240,9 @@ confirm_multivalued_field(Cluster) ->
                     1 -> true;
                     0 -> false
                 end;
-            _ -> false
+            {ok, {search_results, [], _Score, 0}} ->
```

Comment:
I wanted it to fail in that case with a bad match, on the assumption that matching against {ok, {search_results, [], _Score, 0}} is an indication that the data has not yet been indexed.  In that case, we should wait.  Otherwise, we should fail the test.